adminapi/exceptions.py

```diff
@@ -0,0 +1,40 @@
+"""adminapi - Exceptions
+
+Copyright (c) 2019 InnoGames GmbH
+"""
+
+
+class AdminapiException(Exception):
+    """Adminapi exception parten class."""
```

Comment:
It was already names like that. Actually it would be cool if you could have a look at the Exception moving commit on it's own. I'm not sure the ApiError stuff I did makes sense.